src/main/java/nextstep/ladder/domain/ladder/Direction.java

```diff
@@ -0,0 +1,40 @@
+ package nextstep.ladder.domain.ladder;
+
+ import java.util.Objects;
+
+ public class Direction {
```

Comment:
```suggestion
package nextstep.ladder.domain.ladder;

import java.util.Objects;

public class Direction {
```

갑자기 앞에 불필요한 공백이 들어갔네요 👀